src/api_routes_http.rs

```diff
@@ -320,6 +320,14 @@ pub fn config(cfg: &mut web::ServiceConfig, rate_limit: &RateLimitCell) {
           .route("/leave_admin", web::post().to(route_post::<LeaveAdmin>)),
       )
       // Admin Actions
+      .service(
```

Comment:
You should be able to move this service into the admin scope above, and change its name to `/purge` .

Reply:
I was just doing this as you commented. Have pushed the changes.